### PR TITLE
[TabBar] Add a 1 pt bottom divider to MDCTabBar

### PR DESCRIPTION
**Pull request by jmdetloff**
https://www.pivotaltracker.com/story/show/156742602

Before:
screen shot 2018-04-18 at 12 35 45 pm

After:
screen shot 2018-04-18 at 12 35 17 pm

### CODE REVIEW

File: components/Tabs/src/MDCTabBar.m [hunk 1]
```diff
@@ -66,6 +66,9 @@
 /// Title-image padding for bottom navigation bars, in points.
 static const CGFloat kBottomNavigationTitleImagePadding = 3;
 
+/// Height for the bottom divider.
+static const CGFloat kBottomNavigationBarDividerHeight = 1;
```
Comment (jverkoey) [inline components/Tabs/src/MDCTabBar.m:70]: Just to confirm: this is 1pt and not 1px.

**Review by jverkoey — APPROVED** [review]
LGTM but I'm having a hard time seeing the divider in the screenshot. Can you set the tab bar to white and make another screenshot?

**Comment by jverkoey** [comment]
Ah, a tip: you can press Cmd+S to save a screenshot of the screen on the simulator.

**Comment by jmdetloff** [comment]
It is fairly subtle, the baseline opacity is 0.12, but the color is paramater-izable, so here's a screenshot with a grey background and a flat red divider. 

simulator screen shot - iphone x - 2018-04-20 at 14 09 50

**Pull request merged**